src/pages/home/components/Sidebar/Sidebar.module.scss

```
@@ -4,17 +4,49 @@
 .container {
   width: 100%;
   height: 100%;
+  height: 100%;
+  max-height: 80vh;
+
+  flex-direction: column;
 }
 
 .accordionContainer {
-  width: 100%;
-
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   gap: 0.8rem;
 }
 
+.scrollableContent {
+  width: 100%;
+  overflow-y: auto;
+
+  flex: 1;
+
+  padding: 0 5px;
+
+  &::-webkit-scrollbar {
+    width: 4px;
+
+    margin-left: 3px;
```

Nesse caso também utilizar rem 
```suggestion
    margin-left: 0.3rem;
```